ops/services/alerts/app_service_metrics/_var.tf

```diff
@@ -85,6 +85,10 @@ variable "skip_on_weekends" {
   default = false
 }
 
+variable "skip_when_off_hours" {
+  default = false
+}
+
```

Comment:
As a note, the above code has been added to give us a variable we can toggle for other alerts on a per-environment basis. Since the requested changes for this one are on `prod`, we're making them inline with the alert query, since these variables traditionally do not apply at that environment level.